scripts/arangodb-docker-container-setup/docker-container-scripts/test-data/testData.json

```diff
@@ -1,357 +1,404 @@
 {
```

Comment:
Reset this and only override it once @yaanahuu2 has updated his script.

Reply:
On second thought, there's no need. As long as the test passes, you should be able to say 'N' to uploading the test data, in which case the script should be fine. TODO- confirm that this is the case!